standards/core/peer-exchange.md

```diff
@@ -66,26 +66,40 @@ message PeerInfo {
   bytes enr = 1;
 }
 
-message PeerExchangeQuery {
+message PeerExchangeRequest {
   uint64 num_peers = 1;
 }
 
 message PeerExchangeResponse {
   repeated PeerInfo peer_infos = 1;
+  uint32 status = 10;
+  optional string desc = 11;
```

Comment:
Let's use the exact same field names as we have used for the new Lightpush and Filter protocols, namely:
`status_code` and `status_desc`.

Reply:
Ok.